app/models/articles/cached_entity.rb

```diff
@@ -0,0 +1,14 @@
+module Articles
```

Comment:
I had this in `app/services` at first, but it really isn't a service. It's a simple model with a secondary constructor and during the discussions in the `Campaign` model PR ( #9300) people seemed ok with having them in `app/models`.

Reply:
Can't wait for the reorganization of the `services` folder :D